middleware/src/prometheus/prometheus.go

```diff
@@ -84,6 +86,43 @@ func (client *PromClient) VerificationProgress() float64 {
 	}
 	queryResult := gjson.Get(response, "data.result").Array()
 	firstResultValue := queryResult[0].Map()["value"].Array()
-	log.Println("result")
 	return firstResultValue[1].Float()
 }
+
+// GetInt queryies prometheus with the provieded query and returns an int64.
+func (client *PromClient) GetInt(query string) (int64, error) {
+	response, err := client.query(query)
+	if err != nil {
+		log.Println(err.Error())
```

Comment:
`log.Error`?

Reply:
I don't think that exists :) https://golang.org/pkg/log/